test/vinyl-luatest/suite.ini

```diff
@@ -1,4 +1,4 @@
 [default]
 core = luatest
-description = vinyl space engine luatests
+description = vinyl integration tests in luatest
```

Comment:
Please avoid unrelated changes - they complicate review and obscure the history.

Reply:
Fixed.